README.md

```diff
@@ -19,7 +19,30 @@ Note: Spektate dashboard will delete deployments when their corresponding builds
 
 Follow the steps in this [guide](https://github.com/CatalystCode/spk/blob/master/docs/service-introspection-onboarding.md) to onboard a project to use Spektate.
 
-## Dashboard prototype
+## Install on your cluster
+
+The helm chart to use this dashboard is located [here](./chart). There's a Load Balancer provided in the helm chart but it's turned off in `values.yaml` with the setting `externalIP`. Set this to true if you would like to expose the dashboard via a public endpoint.
+
+Substitute values for your configuration and install this dashboard via the command below:
+
+```bash
+cd ./chart
+helm install . --name spektate --set storageAccessKey=<storageAccessKey> --set storageTableName=<storageTableName> --set storagePartitionKey=<storagePartitionKey> --set storageAccountName=<storageAccountName> --set pipelineProject=<pipelineProjectName> --set pipelineOrg=<pipelineOrg> --set pipelineAccessToken=<PipelinePAT> --set manifest=<manifestRepoName> --set manifestAccessToken=<manifestAccessToken> --set githubManifestUsername=<gitHubUserName>  --set sourceRepoAccessToken=<sourceRepoAccessToken>
+```
+
+- `storageAccessKey`: Access key for the storage account
+- `storageTableName`: Table name for the storage account
+- `storagePartitionKey`: Partition key for your configuration, you may want to use project name or some identifier that helps separate unrelated configurations for the purpose of introspection.
+- `storageAccountName`: Storage account name
+- `pipelineProject`: Project name for the pipelines in Azure DevOps
+- `pipelineOrg`: Org name for the pipelines in Azure DevOps
+- `pipelineAccessToken`: Access token for pipelines in Azure DevOps
+- `manifestRepoName`: Manifest repository name
+- `manifestAccessToken`: Access token for the manifest repository
+- `sourceRepoAccessToken`: Access token for the source repository
+- **Note**: If you're using GitHub, add `githubManifestUsername`: Account name under which the manifest repository resides.
```

Comment:
@samiyaakhtar do you mean the the org name in a GitHub repo? For instance microsoft is the org name in github.com/microsoft/bedrock. If so we should change the variable name